website/tutorial/basic/01-project-setup.mdx

```diff
@@ -0,0 +1,130 @@
+---
+id: 01-project-setup
+title: Project Setup
+sidebar_label: Project Setup
+---
+
+In this section, you will learn how to create a basic NodeJS project with TypeScript.
+
+This step will cover the initial NodeJS setup required, basic configuration for TypeScript projects, and how to setup the development scripts.
+
+### Requirements
+
+- Have NodeJS installed on your operation system ([instructions](https://nodejs.org/en/download/package-manager/)). You can use any recent version (12/14/16).
```

Comment:
should we say we recommend to always use the latest version, even if it's not LTS?

Reply:
I would not recommend using Node.js 17 as it still has a few funky behaviors regarding host resolution (at leats I noticed that on Windows).